pymatgen/io/lammps/output.py

```diff
@@ -2,22 +2,7 @@
 # Copyright (c) Pymatgen Development Team.
 # Distributed under the terms of the MIT License.
 
-from __future__ import division, print_function, unicode_literals, \
-    absolute_import
-
-import re
-from io import open
-import os
-
-import numpy as np
-
-from monty.json import MSONable
-
-from pymatgen.core.periodic_table import _pt_data
-from pymatgen.core.structure import Structure
-from pymatgen.core.lattice import Lattice
-from pymatgen.analysis.diffusion_analyzer import DiffusionAnalyzer
-from pymatgen.io.lammps.data import LammpsData, LammpsForceFieldData
+from __future__ import division, print_function, unicode_literals, absolute_import
```

Comment:
are we talking about the same pymatgen that contains the chemenv and io.abinit modules.. ..haha